BioArchLinux/r-rprotobuf/lilac.yaml

```
@@ -0,0 +1,9 @@
+build_prefix: extra-x86_64
+maintainers:
+- github: kiri2002
+repo_depends:
+- r-rcpp
+update_on:
+- regex: RProtoBuf_([\d._-]+).tar.gz
+  source: regex
+  url: https://cran.r-project.org/package=RProtoBuf
```

```suggestion
  url: https://cran.r-project.org/package=RProtoBuf
- source: alpm
  alpm: protobuf
  repo: extra
  provided: libprotobuf.so
  strip_release: true
```
Rebuild config for protobuf

---

Better to use cran instead of regex. @pekkarr now is lilac has the ability to handle the cran archive?

---

Yes, the `cran` source would be better. However the new `rpkgs` source in archlinuxcn/lilac#201 is even more efficient than the `cran` source, but the pr hasn't been merged yet.

---

okay, I see